File: practices/financial_calculator.py
# LL 6th Period Finance Calculator
def calculate_budget(income, rent, utilities, groceries):
    return income - rent - utilities - groceries


income = float(input("Enter your monthly income: "))
rent = float(input("Enter your monthly rent: "))
utilities = float(input("Enter your monthly utilities: "))
groceries = float(input("Enter your monthly groceries: "))
remaining_funds = calculate_budget(income, rent, utilities, groceries)



print(f"Your Monthly Budget is {remaining_funds} ")
print("Total income: $", income)
print("Total expenses: $", rent + utilities + groceries)

File: c_programming/Practices/financial_calculator.c
// LL 6th financial calculator

#include <stdio.h>

int main() {
    float income, rent, utilities, groceries, transportation, savings;

    printf("Monthly income: ");
    scanf("%f", &income);

    printf("Rent: ");
    scanf("%f", &rent);

    printf("Utilities: ");
    scanf("%f", &utilities);

    printf("Groceries: ");
    scanf("%f", &groceries);

    printf("Transportation (Bus pass/car payment): ");
    scanf("%f", &transportation);

    printf("Amount to save: ");
    scanf("%f", &savings);

    float total_expenses = rent + utilities + groceries + transportation;
    float left = income - total_expenses - savings;

    printf("Income: $%f\n", income);
    printf("Total Expenses: $%f\n", total_expenses);
    printf("Savings: $%f\n", savings);
    printf("Left to spend: $%f\n", left);

    return 0;
}